<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>materiel</class>
 <widget class="QDialog" name="materiel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>961</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Gestion des machines</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>891</width>
       <height>421</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupBox_13{
	color: rgb(000, 000, 000);}</string>
     </property>
     <property name="title">
      <string>Machines</string>
     </property>
     <widget class="QLabel" name="label_65">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Algerian</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_65{
	color: rgb(255, 255, 255);}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000001;&quot;&gt;Code de machine :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_66">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Algerian</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000001;&quot;&gt;Type de machine :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="code_mach">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="type_mach">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajout_mach">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">


#ajout_mach:pressed{
	background-color: rgb(85, 0, 0);
color: rgb(255, 255, 255);}</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../geree fournisseurs/projet/i,qge/app.png</normaloff>../../geree fournisseurs/projet/i,qge/app.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_67">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Nouveau dossier/icons8-google-web-recherche-24.png</pixmap>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_mach">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_68">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Algerian</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000001;&quot;&gt;Date :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_mach">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">


#modifier_mach:pressed{
	background-color: rgb(85, 0, 0);
color: rgb(255, 255, 255);}</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../geree fournisseurs/projet/i,qge/B.png</normaloff>../../geree fournisseurs/projet/i,qge/B.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="supp_mach">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">


#supp_mach:pressed{
	background-color: rgb(85, 0, 0);
color: rgb(255, 255, 255);}</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../geree fournisseurs/projet/i,qge/PP.png</normaloff>../../geree fournisseurs/projet/i,qge/PP.png</iconset>
      </property>
     </widget>
     <widget class="QTableView" name="tab_machine">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>451</width>
        <height>281</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_14">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>211</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Algerian</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox_14{
	color: rgb(000, 000, 000);
	
	}</string>
      </property>
      <property name="title">
       <string>Rechercher</string>
      </property>
      <widget class="QComboBox" name="comboBox_machine">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>code</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>type</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>date</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_machine">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="imprimer1">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>350</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf1">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>350</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Gestion des pieces</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>861</width>
       <height>421</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupBox_15{
	color: rgb(000, 000, 000);}</string>
     </property>
     <property name="title">
      <string>Piéce machine</string>
     </property>
     <widget class="QLabel" name="label_69">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Algerian</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000001;&quot;&gt;Code de piéce :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="code_m">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="code_p">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>230</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajout_p">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">


#ajout_p:pressed{
	background-color: rgb(85, 0, 0);
color: rgb(255, 255, 255);}</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../geree fournisseurs/projet/i,qge/app.png</normaloff>../../geree fournisseurs/projet/i,qge/app.png</iconset>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_piece">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_72">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Algerian</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000001;&quot;&gt;Date :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_p">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">


#modifier_p:pressed{
	background-color: rgb(85, 0, 0);
color: rgb(255, 255, 255);}</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../geree fournisseurs/projet/i,qge/B.png</normaloff>../../geree fournisseurs/projet/i,qge/B.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="supp_p">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
#supp_p:pressed{
	background-color: rgb(85, 0, 0);

color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../geree fournisseurs/projet/i,qge/PP.png</normaloff>../../geree fournisseurs/projet/i,qge/PP.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_73">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Algerian</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000001;&quot;&gt;nom:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tab_piece">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>441</width>
        <height>291</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_16">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Algerian</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox_16{
	color: rgb(000, 000, 000);}</string>
      </property>
      <property name="title">
       <string>Rechercher</string>
      </property>
      <widget class="QComboBox" name="comboBox_piece">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Code</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>date</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_piece">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_pdf">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>360</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimer">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>360</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <zorder>code_p</zorder>
     <zorder>label_69</zorder>
     <zorder>code_m</zorder>
     <zorder>ajout_p</zorder>
     <zorder>date_piece</zorder>
     <zorder>label_72</zorder>
     <zorder>modifier_p</zorder>
     <zorder>supp_p</zorder>
     <zorder>label_73</zorder>
     <zorder>nom</zorder>
     <zorder>tab_piece</zorder>
     <zorder>groupBox_16</zorder>
     <zorder>pushButton_pdf</zorder>
     <zorder>imprimer</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-170</x>
     <y>-90</y>
     <width>1271</width>
     <height>731</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Projet/dfd.jpg</pixmap>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
   </widget>
  </widget>
  <zorder>label</zorder>
  <zorder>tabWidget</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
